libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -221,6 +221,13 @@ void AP_SerialManager::init()
                                          AP_SERIALMANAGER_ULANDING_BUFSIZE_RX,
                                          AP_SERIALMANAGER_ULANDING_BUFSIZE_TX);
                     break;
+
+                case SerialProtocol_Volz:
+                    // Note baudrate is hardcoded to 115200
+                	state[i].baud = AP_SERIALMANAGER_VOLZ_BAUD / 1000;   // update baud param in case user looks at it
```

Comment:
why not define AP_SERIALMANAGER_VOLZ_BAUD as 115 instead of dividing by 1000?

Reply:
also, if you look at the driver, 115 and 115200 are treated exactly the same in a switch so I'd suggest removing the /1000

Reply:
changed, i thought about it but tried to follow the rest of the protocols few lines above it.